src/xapiwrapper.js

```diff
@@ -184,7 +182,7 @@ function isDate(date) {
 
         if (verifyxapiversion && testConfig.call(this))
         {
-            window.ADL.XHR_request(this.lrs, this.lrs.endpoint+"about", "GET", null, null,
+            ADL.XHR_request(this.lrs, this.lrs.endpoint+"about", "GET", null, null,
```

Comment:
I have removed all module internal references to the global scope `window` as it's different in different environments. The umd wrapper normalises all of this and it's unnecessary anyway as we already have a reference to the ADL object.